src/bzz.js

```diff
@@ -2,29 +2,28 @@
 
 import debug from 'debug'
 import { BZZ } from 'erebos'
-import { buffer, send } from 'micro'
```

Comment:
Could we use [the `raw()` function from body-parser](https://github.com/expressjs/body-parser#bodyparserrawoptions) here and remove the dependency on micro if we're only using micro for it?